geometry_msgs/Pose2D cell
---
std_msgs/Int16 value
